### PR TITLE
Feature: Gratitude game small changes feedback (KIDS-1566)

### PR DESCRIPTION
**Pull request by TammiLion**
## Summary by CodeRabbit

- **New Features**
	- Enhanced gameplay mechanics for the Guess Secret Word game, allowing dynamic updates to pressed options and improved tracking of correct guesses.

- **Bug Fixes**
	- Fixed the layout of the Rule Card widget for better visual consistency and spacing.

- **Style**
	- Adjusted the padding and height of UI components for an improved user experience.

### CODE REVIEW

File: lib/features/family/features/reflect/bloc/guess_secret_word_cubit.dart [hunk 1]
```diff
@@ -47,6 +48,12 @@ class GuessSecretWordCubit
         );
       }
       _hasSuccess = true;
+      _pressedOptions = [
```
Comment (MaikelStuivenberg) [inline lib/features/family/features/reflect/bloc/guess_secret_word_cubit.dart:51]: Does this not also include the succes button? Or is that one still clickable? :)

**Review by MaikelStuivenberg — APPROVED** [review]

File: lib/features/family/features/reflect/presentation/widgets/rule_card.dart [hunk 2]
```diff
@@ -38,6 +38,7 @@ class RuleCard extends StatelessWidget {
         ),
       ),
       Container(
+        height: 366,
```
Comment (MaikelStuivenberg) [inline lib/features/family/features/reflect/presentation/widgets/rule_card.dart:41]: just to be sure. Should this have a fix height for all rule cards?

File: lib/features/family/features/reflect/bloc/guess_secret_word_cubit.dart (hunk 1 above)
Comment (TammiLion) [reply lib/features/family/features/reflect/bloc/guess_secret_word_cubit.dart:51]: Still clickeable ;) (which is preferred)

File: lib/features/family/features/reflect/presentation/widgets/rule_card.dart (hunk 2 above)
Comment (TammiLion) [reply lib/features/family/features/reflect/presentation/widgets/rule_card.dart:41]: Yes! In the design they are all the same height and by putting it this way the "read out loud" now shows correctly again.

**Pull request merged**